.github/workflows/ci-rust.yml

```
@@ -231,6 +231,15 @@ jobs:
           unzip D:/a/_temp/winfsp-tests.zip -d D:/a/_temp/
           mv 'D:/a/_temp/winfsp-tests-x64.exe' 'C:/Program Files (x86)/WinFsp/bin/'
 
+      - name: Install macfuse
+        if: startsWith(matrix.os, 'macos')
+        shell: bash
+        run: |
+          set -eux
+          brew install --cask macfuse
+          mv /Library/Filesystems/macfuse.fs /Library/Filesystems/fuse.fs
```

```suggestion
          sudo ln -sv /Library/Filesystems/macfuse.fs /Library/Filesystems/fuse.fs
```

Make a symlink instead of moving the lib.

We also need `superuser` permission to do that

---

Nvm we don't need this